www/app.py

```diff
@@ -0,0 +1,27 @@
+from flask import Flask, jsonify, Response
+from os import listdir
+from os.path import isfile, join
+from flask_cors import CORS, cross_origin
```

Comment:
at some point, we'll really really really want to properly take care of our CORS issue instead of just ignoring it. Is there a task for that?